File: c language/question 19.c
// 19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.
#include <stdio.h>
int main(){
int arr[]={1,2,3,4,5};
int len=sizeof(arr)/sizeof(arr[0]);
int cnt=0;
for(int i=0;i<len;i++){
    for(j=i+1;j<len;j++){
        if(arr[i]>arr[j]){
            cnt++;
        }
    }
}
if(cnt==0){
    printf("true");
}
else{
    prinntf("false");
}
return 0;
}

File: Java script/question 19.js
// 19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.
var arr=[1,2,3,4,5]
var len=arr.length
var cnt=0;
for(i=0;i<len;i++){
    for(j=i+1;j<len;j++){
        if(arr[i]>arr[j]){
            cnt++
        }
    }
}
if(cnt==0){
    console.log("true")
}
else{
    console.log("false")
}